selenium_tests/class_roster/class_roster_base_test_case.py

```diff
@@ -0,0 +1,49 @@
+from os.path import abspath, dirname, join
+from django.conf import settings
+
+from selenium_tests.course_admin.course_admin_base_test_case \
+    import CourseAdminBaseTestCase
+from selenium_tests.course_admin.page_objects\
+    .course_admin_dashboard_page_object import CourseAdminDashboardPage
+from selenium_tests.class_roster.page_objects.class_roster_main_page import \
+    MainPageObject
+
+# Common files used for all Class Roster test cases
+CLASS_ROSTER_PERMISSION_ROLES = join(dirname(abspath(__file__)), 'test_data',
+                                     'class_roster_roles_access.xlsx')
+
+
+class ClassRosterBaseTestCase(CourseAdminBaseTestCase):
+
+    def setUp(self):
+        """
+        Redirects browser to test course URL and login to PIN using
+        specified credentials.
+        """
+        super(CourseAdminBaseTestCase, self).setUp()
+
+        # instantiate
+        self.dashboard_page = CourseAdminDashboardPage(self.driver)
+        self.class_roster_page = MainPageObject(self.driver)
+        self.main_page = MainPageObject(self.driver)
```

Comment:
duplicate PO -- we already have it from line 27. We can collapse these two (and fix the references e.g. on line 38 and 49).